libres/lib/enkf/enkf_state.cpp

```diff
@@ -57,405 +53,411 @@
+                "%s, but have observation for this, job will fail.",
+                iens, key);
+            forward_load_context_update_result(load_context, LOAD_FAILURE);
+            if (forward_load_context_accept_messages(load_context)) {
+                char *msg =
+                    util_alloc_sprintf("Failed to load vector: %s", key);
+                forward_load_context_add_message(load_context, msg);
+                free(msg);
+            }
+        }
     }
-  }
 
-  stringlist_free(keys);
+    stringlist_free(keys);
 }
 
-static bool enkf_state_internalize_dynamic_eclipse_results(ensemble_config_type * ens_config,
-                                                           forward_load_context_type * load_context ,
-                                                           const model_config_type * model_config) {
-
-  bool load_summary = ensemble_config_has_impl_type(ens_config, SUMMARY);
-  const run_arg_type * run_arg = forward_load_context_get_run_arg( load_context );
-  const summary_key_matcher_type * matcher = ensemble_config_get_summary_key_matcher(ens_config);
-  const ecl_sum_type * summary = forward_load_context_get_ecl_sum( load_context );
-  int matcher_size = summary_key_matcher_get_size(matcher);
-
-  if (load_summary || matcher_size > 0 || summary) {
-    int load_start = run_arg_get_load_start( run_arg );
-
-    if (load_start == 0) { /* Do not attempt to load the "S0000" summary results. */
-      load_start++;
-    }
+static bool enkf_state_internalize_dynamic_eclipse_results(
+    ensemble_config_type *ens_config, forward_load_context_type *load_context,
+    const model_config_type *model_config) {
+
+    bool load_summary = ensemble_config_has_impl_type(ens_config, SUMMARY);
+    const run_arg_type *run_arg =
+        forward_load_context_get_run_arg(load_context);
+    const summary_key_matcher_type *matcher =
+        ensemble_config_get_summary_key_matcher(ens_config);
+    const ecl_sum_type *summary =
+        forward_load_context_get_ecl_sum(load_context);
+    int matcher_size = summary_key_matcher_get_size(matcher);
+
+    if (load_summary || matcher_size > 0 || summary) {
+        int load_start = run_arg_get_load_start(run_arg);
+
+        if (load_start ==
+            0) { /* Do not attempt to load the "S0000" summary results. */
+            load_start++;
+        }
 
-    {
-      enkf_fs_type * sim_fs = run_arg_get_sim_fs( run_arg );
-      /** OK - now we have actually loaded the ecl_sum instance, or ecl_sum == NULL. */
-      if (summary) {
-        int_vector_type * time_index = __enkf_state_get_time_index(sim_fs, summary);
+        {
+            enkf_fs_type *sim_fs = run_arg_get_sim_fs(run_arg);
+            /** OK - now we have actually loaded the ecl_sum instance, or ecl_sum == NULL. */
+            if (summary) {
+                int_vector_type *time_index =
+                    __enkf_state_get_time_index(sim_fs, summary);
 
-        /*
+                /*
```

Comment:
Indentation of comment now looks weird because the indentation of the code has changed.